dipy/workflows/tracking.py

```diff
@@ -18,6 +20,58 @@
 from dipy.data import default_sphere
 from dipy.direction import DeterministicMaximumDirectionGetter
 from dipy.workflows.multi_io import io_iterator_
+from ipdb import set_trace
+
+
+def det_track_flow(peaks_files, stopping_files, use_sh=False,
```

Comment:
Gab I need to explain you quite a few things. Will try to call you later
this afternoon. Definitely this week!

On Wed, Jun 1, 2016 at 1:02 PM Gabriel Girard <notifications@github.com>
wrote:

> In dipy/workflows/tracking.py
> <https://github.com/matthieudumont/dipy/pull/4#discussion_r65401422>:
>
> > @@ -18,6 +20,58 @@
> >  from dipy.data import default_sphere
> >  from dipy.direction import DeterministicMaximumDirectionGetter
> >  from dipy.workflows.multi_io import io_iterator_
> > +from ipdb import set_trace
> > +
> > +
> > +def det_track_flow(peaks_files, stopping_files, use_sh=False,
>
> how would you best handle tracking from a sh file (e.g. csd_sh.nii.gz)? an
> other flow?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/matthieudumont/dipy/pull/4/files/33703fb2c1c516b63314d4bfa21b6d43fea86b19#r65401422>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAIMhLyelPTjNieD4uA-xZyBKjIBrWbNks5qHbsegaJpZM4Iq-Kz>
> .
>